player/command.c

```diff
@@ -2830,12 +2830,11 @@ static int mp_property_sub_pos(void *ctx, struct m_property *prop,
     return mp_property_generic_option(mpctx, prop, action, arg);
 }
 
-static int mp_property_sub_text(void *ctx, struct m_property *prop,
-                                int action, void *arg)
+static int get_sub_text(void *ctx, struct m_property *prop, int action, void *arg, int sub_index)
```

Comment:
Apologies, I didn't mean for you to make this one line. I meant that `int action...` here should be aligned directly under `void *ctx...` like in old diff.